pyracing/constants.py

```diff
@@ -144,6 +142,10 @@
 URL_MY_RACERS = (mSite + '/GetDriverStatus')
 URL_MEM_DIVISION = (mSite + '/GetMembersDivision')
 
+# iRacing rounds down to the 5 minute mark - DO NOT CHANGE
+class Date:
```

Comment:
This should just be a single method, no need for a class here

Reply:
I made it a class to possibly add more options. An example would be the search queries for team_standings which offer a time lowerbound and time upperbound